src/components/MainTabs/MainTabs.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import MainTabs from "components/MainTabs/MainTabs";
+import { IonReactRouter } from "@ionic/react-router";
+import { expectToBeOnPage } from "test-utils/matchers";
+
+test("renders without crashing", () => {
+  const { container } = render(<MainTabs />, { wrapper: IonReactRouter });
+  expect(container).toBeInTheDocument();
+});
+
+test("visiting /home takes user to Home", () => {
+  window.history.pushState({}, "", "/home");
```

Comment:
In these tests, I've directly used the `window.history` object instead of creating the history myself and passing it as props into `IonReactRouter`. The disadvantage of this approach can be found here: https://github.com/crossroads/app.goodchat.hk/wiki/Testing-React-Router#what-about-windowhistory
In this scenario tests not being truly isolated does not cause isues but would developers be aware of this fact? Btw I couldn't find any way to clear the window.history object. Apparently there are security reasons for this.